### PR TITLE
fix(docs) add missing endpoint for autodoc

### PR DESCRIPTION
**Pull request by mayocream**
### Summary

add missing endpoint for autudoc

### CODE REVIEW

File: autodoc/admin-api/data/admin-api.lua [hunk 1]
```diff
@@ -478,6 +478,72 @@ return {
...
+
+            * `flamegraph`: String-encoded timer-related flamegraph data.
+              You can use [brendangregg/FlameGraph](https://github.com/brendangregg/FlameGraph) to generate flamegraph svgs.
+            * `sys`: List the number of different type of timers.
+              * running: number of running timers.
+              * pending: number of pending timers.
+              * waiting: number of unexpired timers.
+              * total: running + pending + waiting.
+            * `timers.meta`: Program callstack of created timers.
+              `meta.name` is an automatically generated string that stores the location where the creation timer was created.
```
Comment (ADD-SP) [inline autodoc/admin-api/data/admin-api.lua:535]:
```suggestion
            * `timers.meta`: Program callstack of created timers.
              * ` timers.meta.name`: An automatically generated string that stores the location where the creation timer was created.
              * `timers.meta.name`:  Callstack when creating this timer.
```

File: autodoc/admin-api/data/admin-api.lua [hunk 2]
```diff
@@ -478,6 +478,72 @@ return {
...
+                        }
+                    }
+                }
+            }
+            ```
+
+            * `flamegraph`: String-encoded timer-related flamegraph data.
+              You can use [brendangregg/FlameGraph](https://github.com/brendangregg/FlameGraph) to generate flamegraph svgs.
+            * `sys`: List the number of different type of timers.
+              * running: number of running timers.
```
Comment (dndx) [inline autodoc/admin-api/data/admin-api.lua:530]:
```suggestion
              * `running`: number of running timers.
```

Same for the next few lines.

File: autodoc/admin-api/data/admin-api.lua [hunk 3]
```diff
@@ -478,6 +478,72 @@ return {
...
+              * pending: number of pending timers.
+              * waiting: number of unexpired timers.
+              * total: running + pending + waiting.
+            * `timers.meta`: Program callstack of created timers.
+              * `name`: An automatically generated string that stores the location where the creation timer was created.
+              * `callstack`: Callstack string when creating this timer.
```
Comment (dndx) [inline autodoc/admin-api/data/admin-api.lua:536]:
```suggestion
              * `callstack`: Lua call stack string showing where this timer was created.
```

File: autodoc/admin-api/data/admin-api.lua [hunk 4]
```diff
@@ -478,6 +478,72 @@ return {
...
+            * `timers.stats.elapsed_time`: An object that stores the maximum, minimum, average and variance
+              of the time spent on each run of the timer (second).
+            * `timers.stats.runs`: Total number of runs.
+            * `timers.stats.finish`: Total number of successful runs.
+
+            Note: `flamegraph`, `timers.meta` and `timers.stats.elapsed_time` are only available when Kong's log level is `debug`.
```
Comment (dndx) [inline autodoc/admin-api/data/admin-api.lua:542]:
```suggestion
            Note: `flamegraph`, `timers.meta` and `timers.stats.elapsed_time` keys are only available when Kong's `log_level` config is set to `debug`.
```

File: autodoc/admin-api/data/admin-api.lua (hunk 2 above)
Comment (mayocream) [reply autodoc/admin-api/data/admin-api.lua]: Thanks! done!

File: autodoc/admin-api/data/admin-api.lua (hunk 4 above)
Comment (mayocream) [reply autodoc/admin-api/data/admin-api.lua]: Nice!

**Review by dndx — APPROVED** [review]

**Pull request merged**